addons/sales_team/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,34 @@
+---Fields in module 'sales_team'---
+sales_team   / crm.team                 / currency_id (many2one)        : previously in module sale
+sales_team   / crm.team                 / dashboard_graph_group (selection): NEW selection_keys: ['day', 'month', 'user', 'week']
+sales_team   / crm.team                 / dashboard_graph_model (selection): NEW selection_keys: []
+sales_team   / crm.team                 / dashboard_graph_period (selection): NEW selection_keys: ['month', 'week', 'year']
+sales_team   / crm.team                 / favorite_user_ids (many2many) : NEW relation: res.users
+sales_team   / crm.team                 / team_type (selection)         : NEW required: required, selection_keys: ['sales', 'website'], req_default: function
```

Comment:
Is the default assigned team proper for all the possible teams migrated from 10.0?